tomoxlending/lendingstate/trade.go

```diff
@@ -35,6 +35,7 @@ type LendingTrade struct {
 	AutoTopUp              bool           `bson:"autoTopUp" json:"autoTopUp"`
 	LiquidationTime        uint64         `bson:"liquidationTime" json:"liquidationTime"`
 	DepositRate            *big.Int       `bson:"depositRate" json:"depositRate"`
+	LiquidationRate        *big.Int       `bson:"liquidationRate" json:"liquidationRate"`
```

Comment:
I think we also need to add `RecallRate` to the trade item.

Reply:
:+1: